store/gaskv/store.go

```
@@ -6,62 +6,94 @@ import (
+	return NewGStore(parent, gasMeter, gasConfig,
+		func(v []byte) bool { return v == nil },
+		func(v []byte) int { return len(v) },
+	)
+}
+
+type ObjStore = GStore[any]
+
+func NewObjStore(parent types.ObjKVStore, gasMeter types.GasMeter, gasConfig types.GasConfig) *ObjStore {
+	return NewGStore(parent, gasMeter, gasConfig,
+		func(v any) bool { return v == nil },
+		func(v any) int { return ObjectValueLength },
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'v' seems to be unused, consider removing or renaming it as _

(revive)